templates/taxi/driver_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update,Delete" }} Driver</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+    <input class="btn btn-primary" type="submit" value="Submit">
+  </form>
```

Comment:
According to the checklist, the form should have a 'Cancel' button in addition to the 'Submit' button. The 'Cancel' button should lead back to the previous page the user was on. Please add this button to improve the UX of your form.